Source/WebKit/UIProcess/WebFrameProxy.h

```diff
@@ -153,6 +156,8 @@ class WebFrameProxy : public API::ObjectImpl<API::Object::Type::Frame>, public I
 
     WeakPtr<WebPageProxy> m_page;
     Ref<WebProcessProxy> m_process;
+    RefPtr<WebProcessProxy> m_remoteFrameProcess;
```

Comment:
Is this temporary? Seems like m_process should be the remote frame process when the frame is remote?

Reply:
This is the process that has the RemoteFrame.  It's a bad name.  I'm going to rename it to m_parentFrameProcess, but it will be better organized quite soon.